meilidb-http/src/routes/document.rs

```diff
@@ -74,7 +74,7 @@ struct BrowseQuery {
     attributes_to_retrieve: Option<String>,
 }
 
-pub async fn browse_documents(ctx: Context<Data>) -> SResult<Response> {
+pub async fn get_all_documents(ctx: Context<Data>) -> SResult<Response> {
```

Comment:
I don't understand why you renamed this route function like so, it doesn't return all documents but a bunch of them, based on the offset and limit the user specified.

Reply:
I just want to be in agreement with the documentation language. And not to have any conflict with the name "Browse" which has another use in other full-text search engines.